src/config.py

```diff
@@ -49,16 +49,11 @@
 HMMSEARCH_MACHINE_TYPE = os.getenv('HMMSEARCH_MACHINE_TYPE', 'c2-standard-16')
 HHBLITS_MACHINE_TYPE = os.getenv('HMMSEARCH_MACHINE_TYPE', 'c2-standard-16')
 
-MEMORY_LIMIT = os.getenv('MEMORY_LIMIT', '32')
```

Comment:
Could we keep the same set of env variables for the 2 different tasks (predict and relax)?